applications/GeoMechanicsApplication/tests/test_settlement_workflow/ProjectParameters_stage4.json

```diff
@@ -93,8 +93,8 @@
                         "node_output":         false,
                         "skin_output":         false,
                         "plane_output":        [],
-                        "nodal_results":       ["DISPLACEMENT","INCREMENTAL_DISPLACEMENT","TOTAL_DISPLACEMENT","WATER_PRESSURE","VOLUME_ACCELERATION"],
-                        "gauss_point_results": ["GREEN_LAGRANGE_STRAIN_TENSOR","ENGINEERING_STRAIN_TENSOR","CAUCHY_STRESS_TENSOR","TOTAL_STRESS_TENSOR","VON_MISES_STRESS","FLUID_FLUX_VECTOR","HYDRAULIC_HEAD"]
+                        "nodal_results":       ["DISPLACEMENT","INCREMENTAL_DISPLACEMENT","TOTAL_DISPLACEMENT","WATER_PRESSURE","VOLUME_ACCELERATION", "CAUCHY_STRESS_TENSOR","TOTAL_STRESS_TENSOR"],
+                        "gauss_point_results": ["GREEN_LAGRANGE_STRAIN_TENSOR","ENGINEERING_STRAIN_TENSOR","VON_MISES_STRESS","FLUID_FLUX_VECTOR","HYDRAULIC_HEAD", "CAUCHY_STRESS_TENSOR","TOTAL_STRESS_TENSOR"]
```

Comment:
Question: what was the reason to change the order of the requested integration point results?

Reply:
No specific reason, I've just been trying out things with these parameters, I moved it back to how it was to avoid confusion